xbmc/cores/VideoPlayer/VideoRenderers/OverlayRendererDX.cpp

```diff
@@ -249,9 +249,11 @@ COverlayImageDX::COverlayImageDX(CDVDOverlayImage* o)
 #endif
     {
       /* render aligned to screen to avoid cropping problems */
-      m_align  = ALIGN_SCREEN;
+      m_align = ALIGN_SCREEN_AR;
```

Comment:
I know this is not related to your changes but since you're touching this, can you remove the code block above in a second commit? Dead code, not used (#if 0)